### PR TITLE
feat(forc): add "organization" field in forc manifest

### LINKED ISSUE #607: Add organization field to manifest
One issue with Cargo's crate namespacing is that there's no concept of organizations. This means all crates are all in the same namespace, and if a crate name is taken, then it's taken.

Proposal: add an `organization` field to the Forc manifest file, such that projects can be published under both an org scope and have a project name.

### PR DESCRIPTION
**Pull request by vnepveu**
This PR adds the "organization" field in the `Forc.toml` manifest file. 
It closes #607.

### CODE REVIEW

**Comment by vnepveu** [comment]
Overlooked the test harness, working on a fix

File: forc/src/utils/defaults.rs [hunk 1]
```diff
@@ -2,19 +2,21 @@
 /// the chance to insert some helpful comments and nicer formatting.
 pub(crate) fn default_manifest(project_name: &str) -> String {
     let real_name = whoami::realname();
+    let orga_name = "organizationName";
 
     format!(
         r#"[project]
 name = "{}"
 author = "{}"
+organization = "{}"
```
Comment (adlerjohn) [inline forc/src/utils/defaults.rs:11]: Actually I think this should be removed from the default manifest file generated with `init.` It's not even being filled in with a reasonable value. The default manifest file doesn't _have_ to include every single possible field.

File: forc/src/utils/manifest.rs [hunk 2]
```diff
@@ -109,8 +111,9 @@ fn try_parse_contract_id(raw: &Option<String>) -> Result<fuel_tx::ContractId, St
             })?
             .as_slice(),
     )
-    .unwrap())
+        .unwrap())
```
Comment (adlerjohn) [inline forc/src/utils/manifest.rs:114]: Hmm. Why is this formatting different? The code should have been formatted with `cargo fmt`, no?

Comment (vnepveu) [reply forc/src/utils/manifest.rs:114]: Weird because I did run `cargo fmt`, and the fmt CI check is marked as passing.

Comment (vnepveu) [reply forc/src/utils/manifest.rs:114]: Ok the issue was that I didn't run it with the `--all` flag, thanks for pointing that out

File: forc/src/utils/defaults.rs (hunk 1 above)
Comment (vnepveu) [reply forc/src/utils/defaults.rs]: Done

**Review by sezna — APPROVED** [review]

**Review by adlerjohn — CHANGES_REQUESTED** [review]
Revert changes to lockfile

**Review by adlerjohn — APPROVED** [review]

**Review by digorithm — APPROVED** [review]

**Pull request merged**